R/utilities.R

```diff
@@ -842,11 +843,12 @@ maf_to_custom_track = function(maf_data,
 #' @import tidyverse config
 #'
 #' @examples
-#' everything_collated = collate_results(join_with_full_metadata = TRUE)
+#' everything_collated = collate_results(join_with_metadata = TRUE)
 #'
 collate_results = function(sample_table,
                            write_to_file = FALSE,
-                           join_with_full_metadata = FALSE,
+                           join_with_metadata = FALSE,
+                           user_provided_metadata,
```

Comment:
We might also name this `these_samples_metadata` similar to other functions, so it is consistent throughout